e2e/features/record-grant-change-hearing-centre.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Record grant update hearing requirements
+
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+
+  @record-grant-change-hearing-centre @RIA-2634
+  Scenario: record grant change hearing centre
+
+    Then I click the `Overview` tab
+    And I should only see the `caseOfficer_awaitingRespondentEvidence` case progress image
+
+    And I click the `Applications` tab
+    And I click the `Record an application` link
+    Then I am on the `Record an application` page
+    And I add an item to the `Application email` collection
+    And within the `Application email` collection's first item, I upload `{@test.doc}` for the field without a label
+    And I click the `The legal representative` label
+    And I select `Change hearing centre` for the `Type of application` field
+    And I type `change hearing centre reason` for the `Reason for application` field
+    And I select `{$TODAY-3|DD-MM-YYYY}` for the `Date application was made` field
+    And I click the `Granted` label
+    And I type `some application decision reason` for the `Reasons for decision` field
+
+    Then the `Continue` button is enabled
+
+    When I click the `Continue` button
+    Then I am on the `Check your answers` page
+    And I should see `test.doc` in the `Application email` field
+    And I should see `The legal representative` in the `Application from` field
+    And I should see `Change hearing centre` in the `Type of application` field
+    And I should see `change hearing centre reason` in the `Reason for application` field
+    And I should see `{$TODAY-3|D MMM YYYY}` in the `Date application was made` field
+    And I should see `Granted` in the `Application decision` field
+    And I should see `some application decision reason` in the `Reasons for decision` field
+
+    When I click the `Record application` button
+    Then I should see the text `You have recorded an application`
+    And I should see the text `What happens next`
+    And I should see the text `You must now change the designated hearing centre based on the new information provided in the application. The application decision is available to view in the Application tab.`
+
+    When I click the `Close and Return to case details` button
+    Then I should see an alert confirming the case `has been updated with event: Record an application`
+
+    And I click the `Overview` tab
+    And I should only see the `caseOfficer_changeHearingCentre` case progress image
+    And I should see the text `Do this next`
+    And I should see the text `You must now change the designated hearing centre based on the new information provided in the application. The application decision has been recorded and is available in the applications tab.`
```

Comment:
I think here we need to assert the Application Tab as well and I don't think so we need separate Feature for this, we can add this scenario in @record-grant-update-hearing-requirements